owlkettle/gtk.nim

```diff
@@ -401,8 +416,8 @@ proc g_application_run*(app: GApplication, argc: cint, argv: cstringArray): cint
 # Gio.File
 proc g_file_new_for_path*(path: cstring): GFile
 proc g_file_read*(file: GFile, cancelable: pointer, error: ptr GError): GInputStream
-proc g_file_get_path*(file: GFile): cstring
-proc g_file_get_uri*(file: GFile): cstring
+proc g_file_get_path*(file: GFile): OwnedGtkString
```

Comment:
I managed to test g_file_get_path, but not g_file_get_uri because it gets used nowhere.
`gtk_text_buffer_get_text` I couldn't test either because while it gets used in examples/widgets/text_view.nim, it appears to never actually free the string in the textbuffer, ever. Or this proc is never called in there, I don't quite know

Reply:
Sounds good!